include/swift/AST/DiagnosticsParse.def

```diff
@@ -2019,5 +2019,8 @@ ERROR(macro_expansion_expr_expected_macro_identifier,none,
 ERROR(macro_expansion_decl_expected_macro_identifier,none,
       "expected a macro identifier for a pound literal declaration", ())
 
+ERROR(parser_round_trip_error,none,
+      "source file did not round-trip through the Swift Swift parser", ())
```

Comment:
Is `Swift Swift parser` intentional? It sounds confusing to me. 

I'd use `SwiftParser` or `SwiftParser written in Swift` to refer to the new parser.